setup.cfg

```diff
@@ -1,3 +1,61 @@
-[pytest]
-# notebooks to skip running; one case insensitive re to match per line
-skip_run = ^.*JSONInit\.ipynb$
+[metadata]
+name = parambokeh
+version = attr: param.version.get_setupcfg_version
+description = ParamBokeh provides an easy way to generate a UI for param based classes in the notebook or on bokeh server.
+long_description = file: README.md
+long_description_content_type = text/markdown
+license = BSD 3-Clause License
+license_file = LICENSE.txt
+classifiers =
+    License :: OSI Approved :: BSD License
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Development Status :: 4 - Beta
+author = PyViz
+author_email = holoviews@gmail.com
+maintainer = PyViz
+maintainer_email = holoviews@gmail.com
+url = https://ioam.github.io/parambokeh
+project_urls =
+    Bug Tracker = https://github.com/ioam/parambokeh/issues
+    Documentation = https://ioam.github.io/parambokeh
```

Comment:
Not parambokeh.pyviz.org?

Reply:
I could change this. I also could change it in doc/conf to be the canonical url (one day, doc/conf will get various info from elsewhere to avoid duplication). However, I personally don't understand the consequences of not using https (search rankings, user warnings - e.g. https://security.googleblog.com/2018/02/a-secure-web-is-here-to-stay.html). So to confirm: I should still change it?

Reply:
Is there a reason not to use https://parambokeh.pyviz.org/?

Reply:
No - great (I'd just typed in parambokeh.pyviz.org and received the 'not secure' note). So I'll change this.

Reply:
@philippjfr, do you happen to know why the same isn't true of pyviz.org? I.e. while https://parambokeh.pyviz.org works, https://pyviz.org doesn't seem to. I've lost track of who's doing what/who owns which domain...